department-of-veterans-affairs/va.gov-cms
# VACMS-7519 claro paragraphs functionality

## Description

Closes #7519. Adds Drupal core patch that fixes table construction in the Claro theme. This is necessary for paragraphs, as the paragraphs module shoves some data into table headers for functionality (edit/collapse buttons). Cleans up the focus state for paragraph dropdown button actions. This PR also ports over the rest of the code from `vagovadmin.theme`, which has some paragraph and vet services alters. 


## Testing done
- verified paragraph functionality (Collapse/Edit all) works as expected in vagovclaro and vagovadmin
- verified paragraph dropdown actions have a visible focus state in vagovclaro
- aligned paragraph draggable handles to the top, as they are in vagovadmin.
- verified Vet Center Services are receiving the correct form alters now

## Screenshots
updated correct functionality in claro theme
![Screen Shot 2022-01-18 at 9 01 06 AM](https://user-images.githubusercontent.com/11279744/149987385-787f01cc-0f71-45e8-850c-9f361b9373e5.png)
![Screen Shot 2022-01-18 at 9 01 18 AM](https://user-images.githubusercontent.com/11279744/149987393-5ce8e334-592a-4c74-82df-6a1fc89c35c4.png)

unchanged functionality in current theme
![Screen Shot 2022-01-18 at 9 08 29 AM](https://user-images.githubusercontent.com/11279744/149987394-247b90a4-a48d-4af9-bb4c-96411b23df2b.png)
![Screen Shot 2022-01-18 at 9 09 03 AM](https://user-images.githubusercontent.com/11279744/149987399-d02cc2bf-c593-4a5e-aa94-32f5e4ed23d1.png)
![Screen Shot 2022-01-18 at 9 09 14 AM](https://user-images.githubusercontent.com/11279744/149987403-b56899f5-8b6d-4f14-a34d-587ec03a1b50.png)

claro paragraph actions focus state:
![Screen Shot 2022-01-18 at 9 23 11 AM](https://user-images.githubusercontent.com/11279744/149987409-cd97a1c7-a406-42f1-8402-984c90f88fd7.png)

vet center services list fix (from ported .theme code):
![Screen Shot 2022-01-18 at 9 18 39 AM](https://user-images.githubusercontent.com/11279744/149987406-abc4d86b-c75c-4e79-b427-8d27014d4f26.png)


## QA steps

as admin user,

1. visit an existing centralized content node and/or cms knowledge base node, see paragraphs functionality works as expected in vagovclaro and vagovadmin.
2. in vagovclaro, click on paragraphs dropdown dots and hit tab key to see a visible focus state

as a vet center editor (witt.cook@va.gov): 
1. visit `/node/3877/edit` and see that the services list has the correct form appearance (sorted with required first, no handles, redundant part of label removed)


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
